src/Trace/Integrations/Sqs.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace OpenCensus\Trace\Integrations;
+
+class Sqs implements IntegrationInterface
+{
+
+    public static function load()
+    {
+        if (!extension_loaded('opencensus')) {
+            trigger_error('opencensus extension required to load Sqs integrations.', E_USER_WARNING);
+        }
+
+        opencensus_trace_method('Aws\AwsClient', 'execute', function ($command, $args) {
+            if (get_class($command) === 'Aws\Sqs\SqsClient') {
+                $internalFields = Sqs::formatArguments($command, $args);
+                return [
+                    'attributes' => $internalFields,
+                    'kind' => 'client',
+                    'name' => 'Sqs ' . $args->getName(),
+                    'sameProcessAsParentSpan' => false
+                ];
+            }
+        });
+
+        opencensus_trace_method('Aws\Sqs\SqsClient', '__construct', function ($command, $args) {
```

Comment:
add unit test for the integration added pls.

Reply:
added